mp4parser.py

```diff
@@ -136,11 +137,24 @@ def int(self, n: int) -> int:
 		return int.from_bytes(self.read(n), 'big')
 
 	def string(self, encoding='utf-8') -> str:
+		"""Parse a C-like string."""
 		data = self.peek()
 		if (size := data.tobytes().find(b'\0')) == -1:
 			raise EOFError('EOF while reading string')
 		self.pos += size + 1
-		return data[:size].tobytes().decode(encoding)
+		return data[:size].tobytes().decode(encoding, errors='replace')
```

Comment:
hmm I'm not sure if I like silently replacing errors, I'd prefer the whole thing to very visible fail if we ever find non-utf-8 in it, unless you think it really happens in the wild often enough for it not to be a rounding error

Reply:
iirc I've seen it happen while working on tx3g support... but I can leave it out and change it later with a test vector.

I don't like it either, and I don't like that there is seemingly no good option.